Dockerfile

```diff
@@ -1,4 +1,8 @@
-FROM golang:1.18-alpine as builder
+# If you change this value, please change it in the following files as well:
+# /Dockerfile
+# /tools/Dockerfile
+# /.github/workflows/main.yml
+FROM golang:1.19.13-alpine3.18 as builder
```

Comment:
Any specific reason for also pinning the alpine version? Or just because the default for the Go 1.19 image is an older one?
Looks like if we go to Go 1.21.6 we get alpine 3.19 by default, so I'd suggest just using the `golang:1.21.6-alpine` image here.